Resources/doc/1-setting_up_the_bundle.rst

```diff
@@ -1,8 +1,18 @@
 Step 1: Setting up the bundle
 =============================
 
-A) Install FOSRestBundle
-------------------------
+A) Download the Bundle
+----------------------
+
+Open a command console, enter your project directory and execute the
+following command to download the latest stable version of this bundle:
+
+.. code-block:: bash
+
+    $ composer require friendsofsymfony/rest-bundle "~1.4"
```

Comment:
Yes, but we (sf doc team) recommend to use the cersion in the docs, as you want users to install the version for which the documentation is created (otherwise, 1.0 docs would have a command that installs 2.0)